### ISSUE TITLE
Pretrain with MIMIC-CXR Val Loss

### ISSUE THREAD

**Issue by SinuoWang** [marshuang80/gloria#22]
Hi,

I am pretraining the model with the MIMIC-CXR JPG dataset, but the local alignment validation loss is kinda weird. Is this normal when you pretrain GLoRIA with MIMIC? 

Thanks in advance!!
![loss](https://github.com/marshuang80/gloria/assets/76032182/830b303f-df3d-4527-80fc-59f3271b9649)

**Comment by SaikaOfficial** [comment]
I am training on this dataset too, but your val_loss much bertter than me. Can u share some of your steps of training on it?

**Comment by SinuoWang** [comment]
> I am training on this dataset too, but your val_loss much bertter than me. Can u share some of your steps of training on it?

Maybe you should check whether you modified the img transforms. It's good to use the tf that is specifically for MIMIC not Chexpert.

But you don't need to worry too much about the local val loss. I tested this model on some downstream tasks and got normal results, even the local val loss is super weird

**Comment by SaikaOfficial** [comment]
> > I am training on this dataset too, but your val_loss much bertter than me. Can u share some of your steps of training on it?
> 
> Maybe you should check whether you modified the img transforms. It's good to use the tf that is specifically for MIMIC not Chexpert.
> 
> But you don't need to worry too much about the local val loss. I tested this model on some downstream tasks and got normal results, even the local val loss is super weird 
> 
> 

Sorry, I am not so familiar with these transforms. What is the tf means in your experiment. I also met some wired thing like the downstream tasks like binary classification on pnumonia troax always shows auroc=0.5 and any pic going through the model will become an all-none-tensor. The Adam lr never changed. I don't know is that normal.

**Comment by ZiyangZhang0511** [comment]
> Hi,
> 
> I am pretraining the model with the MIMIC-CXR JPG dataset, but the local alignment validation loss is kinda weird. Is this normal when you pretrain GLoRIA with MIMIC?
> 
> Thanks in advance!! ![loss](https://private-user-images.githubusercontent.com/76032182/299009074-830b303f-df3d-4527-80fc-59f3271b9649.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2ODI3MjAsIm5iZiI6MTcyODY4MjQyMCwicGF0aCI6Ii83NjAzMjE4Mi8yOTkwMDkwNzQtODMwYjMwM2YtZGYzZC00NTI3LTgwZmMtNTlmMzI3MWI5NjQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDExVDIxMzM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0OGM0OTdmZjBmNTk5NTQ5NGUxNWY3NGYyMjk5ZDM5ZmM2M2IwZmVhNDdjZWFhNTlhNmMwMjZjNDI4NWVmY2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GqLLTQjhtXnuOD0TglNVeGNT511Ka0j-amw2_JI5Yc8)

I met the same issues as you. The lowest val loss is 3.4 but train loss can be optimized down to 0.09. This lead to quiet low performance of image-text retreival on test 5x200 dataset. Do you know how to improve R@K or P@K to achieve comparable results that were reported in papers? Thank you in advance.
